### ISSUE TITLE
Bipolar Disorder Ontology

### ISSUE THREAD

**Issue by phwegner** [OBOFoundry/OBOFoundry.github.io#1811]
<a href="https://obofoundry.org/obo-nor.github.io/dashboard/bpdo/dashboard.html"><img src="https://img.shields.io/endpoint?url=https%3A%2F%2Fraw.githubusercontent.com%2FOBOFoundry%2Fobo-nor.github.io%2Fmaster%2Fdashboard%2Fbpdo%2Fdashboard-qc-badge.json" /></a>

## Ontology title
Bipolar Disorder Ontology

## Requested ID space
BPDO

## Ontology location
https://github.com/SCAI-BIO/bipolar-disorder-ontology/blob/main/releases/1.0.0/BPDO.owl

## Contact person
Name: Alpha Tom Kodamullil
Email address: alpha.tom.kodamullil@scai.fraunhofer.de
GitHub username: akodamullil

## Issue tracker
https://github.com/SCAI-BIO/bipolar-disorder-ontology/issues
## Version Controlled Repository
https://github.com/SCAI-BIO/bipolar-disorder-ontology
## Ontology license

- [ ] CC0 (public domain)
- [X] CC-BY (version 3 or later)
- [ ] Other: *please specify*

## Available ontology formats
.owl

## What domain is the ontology intended to cover?
Bipolar Disorder 

## Related OBO Foundry ontologies
DOID, BFO

## Intended use/related projects
A disease ontology to classify Bipolar Disorder and calculate semantic similarities in the domain of Bipolar Disorder, Annotation of medical texts

## Data source
Several sources from domain experts assembled by Fraunhofer SCAI and Computational Biomodels (CBM)

## OBO Foundry pre-registration checklist
Created by the team for applied semantics at Fraunhofer SCAI
To be considered for inclusion in the OBO Foundry, an ontology must meet certain requirements, as described in in [the registration process instructions](http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) and the [registration review checklist](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md). To ensure you are aware of some of its key points, please review the checklist below.

You can either check a box by submitted the request first and then using the GitHub interface, or replacing the `- [ ]` by `- [X]` in the following.

- [X] I have read and understood [the registration process instructions](http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) and the [registration checklist](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md) 
- [X] There is no other ontology in the OBO Foundry which would be an appropriate place for my terms. If there was, I have contacted the editors, and we decided in mutual agreement that a separate ontology is more appropriate.
- [X] My ontology has a specific release file with a version IRI and a dc:license annotation, serialised in RDF/XML.
- [X] I understand that term definitions, while not mandatory, are key to understanding the intentions of a term especially when the ontology is used in curation. I made sure that a reasonable majority of terms in my ontology have definitions, using the [IAO:0000115](http://www.ontobee.org/ontology/IAO?iri=http://purl.obolibrary.org/obo/IAO_0000115) property.
- [X] For every term in my ontology, I checked whether another OBO Foundry ontology has one with the same meaning. If so, I re-used that term directly (not by cross-reference, by directly using the IRI).
- [X] For all relationship properties (Object and Data Property) I checked whether [RO](http://www.ontobee.org/ontology/catalog/RO?iri=http://www.w3.org/2002/07/owl%23ObjectProperty) includes an appropriate one. I understand that aligning with RO is an essential part of the overall alignment between OBO ontologies!
- [X] For the selection of appropriate annotation properties, I looked at [OMO](http://www.ontobee.org/ontology/catalog/OMO?iri=http://www.w3.org/2002/07/owl%23AnnotationProperty) first. I understand that aligning ontology metadata and term-level metadata is essential for cross-integration of OBO ontologies. 
- [X] If I was not sure about the meaning of any of the checkboxes above, I have consulted with a member of the OBO Foundry for advice.
- [X] The requested ID space does not conflict with another ID space found in other registries such as BioPortal and the Bioregistry, see [here](https://obofoundry.org/id-policy) for a complete list.

## Metadata

Please fill in the following metadata record which will be used by the OBO Foundry website. Note that the values shown are just examples, for example `yourfourletterid` could be something like `aism`, `cohm`, `mondo` (it does not have to be four letters). `your_domain_like_for_example_anatomy` could be simply `anatomy`, and the license should be whatever your actual license is. An example can be found [here](https://github.com/OBOFoundry/OBOFoundry.github.io/edit/master/ontology/amphx.md), but you really only need to fill in the metadata mentioned here.

```
id: BPDO
title: Bipolar Disorder Ontology
contact:
  email: alpha.tom.kodamullil@scai.fraunhofer.de
  label: Alpha Tom Kodamullil
description: The Bipolar Disorder ontology includes bipolar disorder-related concepts, terms and related synonyms, collected from wide variety of sources such us publications and websites.
domain: Mental Disorder
homepage: https://github.com/SCAI-BIO/bipolar-disorder-ontology
products:
  - id: BPDO.owl
  - 
dependencies:
  - id: BFO
tracker: https://github.com/SCAI-BIO/bipolar-disorder-ontology/issues
license:
  url: http://creativecommons.org/licenses/by/4.0/
  label: CC-BY
usages:
  - user: - 
    description: - 
```

**Comment by matentzn** [comment]
Thank you for your submission! 

To be _considered_ for acceptance, no dashboard checks aside from usages can be red.

https://obofoundry.org/obo-nor.github.io/dashboard/bpdo/dashboard.html

Please let me know when you are done and I will re-run the checks!

**Comment by matentzn** [comment]
@hoganwr will review this submission and provide feedback.

**Comment by hoganwr** [comment]
With respect to the OBO Dashboard report, please note two issues:
1. The version IRI is not in the recommended format.  Normally we have either an ISO8601 date format version (e.g., 2022-04-27) or for numeric version identifiers, they are of the format NN.n where the n's are integers.  I think the extra .0 is causing robot to flag your version IRI (you have 1.0.0, we would recommend just having 1.0 or better still, a date in ISO8601 format).
2. The ontology should have a dc:title annotation with the formal name of the ontology. I'm assuming the value of the title annotation would be "Bipolar Disorder Ontology"

**Comment by matentzn** [comment]
In case you are wondering what semver is, this is it: https://semver.org/

The important thing is that the version IRI should resolve. So when I try to load "https://bio.scai.fraunhofer.de/ontology/BPDO/1.0.0", it should resolve to the specific ontology version indicated.

Are you @phwegner are you willing to move to the recommended OBO IRI scheme for Version IRIs, or is there a technical reason that prevents you from doing this? If this is not possible, can your versionIRI be made resolvable?

Required submitter actions for now:

- [ ] respond to question on versionIRI
- [ ] @phwegner to add dc:title to ontology
- [ ] @phwegner to migrate all temporary "Fraunhofer" IRIs to OBO IRIs (https://bio.scai.fraunhofer.de/ontology/BPDO#bipolar_II_disorder,_most_recent_episode_major_depressive_with_melancholic_features --> http://purl.obolibrary.org/obo/BPDO_0000001)

There are a few open question our domain expert reviewer is still gaining clarity on, we will feed this back to you asap.

**Comment by BideZ** [comment]
@matentzn @hoganwr Thank very much for your feedback. 

Regarding the version IRI: We would like to stick with the semver principle and keep the IRI due to internal reasons. We will make the IRI resolve in the next update. 

We will also address the other issues as soon as possible., and I will leave a comment when the update is pushed.

**Comment by matentzn** [comment]
Thank you! @BideZ

**Comment by hoganwr** [comment]
That's my understanding.  We are waiting on them to fix IRIs among other
things.

**Comment by phwegner** [comment]
@matentzn First of all please excuse the very late reply due to the current workload this submission, unfortunately, could not be prioritized. But we now added a title. Changed the version IRI format and changed the temporary fraunhofer iris according to your prior comment. 

Are there any other issues that we need to address?

**Comment by matentzn** [comment]
@phwegner thank you. Some more issues:

```
<owl:Ontology rdf:about="http://purl.obolibrary.org/obo/BPDO_">
        <owl:versionIRI rdf:resource="http://purl.obolibrary.org/obo/BPDO/2022-06-17"/>
```

- You have `http://purl.obolibrary.org/obo/BPDO_` as ontology IRI. This is not an ontology IRI, this is a URI base for your terms.. It should be `http://purl.obolibrary.org/obo/bpdo.owl`
- You have `http://purl.obolibrary.org/obo/BPDO/2022-06-17` as the version IRI. This is not a legal version IRI, it should be: `http://purl.obolibrary.org/obo/bpdo/releases/2022-06-17/bpdo.owl`, see https://obofoundry.org/principles/fp-004-versioning.html

Not dashboard related but some other issues I can see glancing at your file (would come out in the technical review anyways):

- [ ] You have a range of broken URLs in BPDO: `http://purl.obolibrary.org/obo/BPDO#oboInOwl:hasDbXRef`
- [ ] You are using some really strange prefixes:
    ```
         xmlns:BPDO2="http://purl.obolibrary.org/obo/BPDO#oboInOwl:"
         xmlns:oboI2="http://www.geneontology.org/formats/oboInOWL#"
         xmlns:x_0.1="http://xmlns.com/foaf/0.1/"
         xmlns:x_1.1="http://purl.org/dc/elements/1.1/"
    ```
    The first two are just wrong (wrong URI expansions!), the latter two are really weird, should be `foaf:` and `dce:`.
- [ ] Some really strange things happened with your imported terms, like from BCGO:
     ```
      <BPDO:Author>Abish</BPDO:Author>
       <BPDO:CommentOnDefinition>Modified definition</BPDO:CommentOnDefinition>
       <BPDO:fromPubMed></BPDO:fromPubMed>
       <BPDO2:hasDbXRef>https://www.ncbi.nlm.nih.gov/pubmed/16158906</BPDO2:hasDbXRef>
     ```
     These should all be using OBO standard properties.

**Comment by phwegner** [comment]
@matentzn Thank you for the fast response. I will address these things and update you asap.

**Comment by BideZ** [comment]
Thank you @matentzn for the feedback.

- We have now firstly changed the ontology IRI and the version IRI to 
http://purl.obolibrary.org/obo/bpdo.owl and http://purl.obolibrary.org/obo/bpdo/releases/2022-06-17/bpdo.owl, respectively.


- Besides, the broken URIs have been fixed, including the annotation properties:
http://purl.obolibrary.org/obo/bpdo#oboInOwl:hasDbXRef
http://purl.obolibrary.org/obo/bpdo#oboInOwl:hasExactSynonym
http://purl.obolibrary.org/obo/bpdo#oboInOwl:hasRelatedSynonym
http://purl.obolibrary.org/obo/IAO_0000115 is now used instead of http://purl.obolibrary.org/obo/BPDO#oboInOwl:hasDefinition

- Furthermore, we have also made the URL format standard for original BPDO terms (http://purl.obolibrary.org/obo/BPDO_XXXXXXXX).


> Some really strange things happened with your imported terms, like from BCGO:

>  <BPDO:Author>Abish</BPDO:Author>
>   <BPDO:CommentOnDefinition>Modified definition</BPDO:CommentOnDefinition>
>   <BPDO:fromPubMed></BPDO:fromPubMed>

> These should all be using OBO standard properties.

This has happened due to usage of some original annotation properties. We would like to stick with these ones since no equivalent OBO terms can be used for the expression. 

Note: 
>   <BPDO2:hasDbXRef>https://www.ncbi.nlm.nih.gov/pubmed/16158906</BPDO2:hasDbXRef>

this has now been changed to 
<oboInOwl:hasDbXref>https://www.ncbi.nlm.nih.gov/pubmed/16158906</oboInOwl:hasDbXref> through using the OBO standard property.

**Comment by matentzn** [comment]
Currently something is going wrong with your ontology:

https://obofoundry.org/obo-nor.github.io/dashboard/index.html

Please make PR on dashboard metadata with the most up to date info:

https://github.com/OBOFoundry/obo-nor.github.io/blob/master/dashboard-config.yml

**Comment by matentzn** [comment]
@BideZ I have updated the board! Thank you for your changes. Please look around for further errors:

https://obofoundry.org/obo-nor.github.io/dashboard/index.html

**Comment by BideZ** [comment]
@matentzn Sorry for the late response. Please have a look at the latest changes regarding the issues mentioned on the board

**Comment by matentzn** [comment]
Ok, dashboard looks ok now. https://obofoundry.org/obo-nor.github.io/dashboard/bpdo/dashboard.html

@hoganwr will make sure all checkboxes in this issue are checked, and inform you of the next steps.

**Comment by BideZ** [comment]
@wdduncan 
Thank you for the feedback. 
We have now: 
- Added object properties that will be used as text mining bins.
- Fixed the aforementioned hierarchy issue and placed the terms under corresponding parent terms.
- Added mental disorder class from MONDO
- social adversity is now placed under 'pathological mental process'

**Comment by balhoff** [comment]
@hoganwr we checked on the status of this request on the OBO Ops meeting — can you follow up on the latest changes?

**Comment by hoganwr** [comment]
This request was revisited at the OFOC call on March 21. The unanimous (or at least nearly so) decision of the group was to deny this request. Note that although we're closing out this request, it is possible to submit a new request for the same ontology / namespace prefix in the future.

We recommend that you do more to familiarize yourselves with OBO principles and how they get put into action.

Specifically:
1. You still have a number of malformed IRIs. The list is below. Mostly they are annotation properties. 
2. Furthermore, many of these annotation properties mention external non-ontological artifacts including medical terminologies like ICD, SNOMED, and MeSH. You should review how existing OBO ontologies reference such content.
3. Futhermore, the labels of these annotation properties do not follow OBO naming conventions
4. You did add eight object properties, all imported from ADO (Alzheimer's Disease Ontology). Note that the ROBOT report is increasingly flagging object properties that either are not in RO or are not subproperties of RO object properties. None of these are from RO or subproprties of RO properties.
5. You have injected axioms onto classes in other ontologies, another issue that OBO is trying to avoid. For example you say for the OGMS:treatment class: subClassOf ('planned process' and (ADO_00000002 some 'bipolar disorder'))
6. Another problem with this axiom is that it is not true that every treatment is a treatment for bipolar disorder
7. Similar issues to #5-6 with SYMP:symptom. Not every symptom is a symptom of bipolar disorder
8. Similar issues to #5-6 with NCIT:Biological Model. Not every such model is a model of bipolar disorder
9. Similar issues to #5-6 with NCIT:Cognitive Assessment. Not every cognitive assessment is a "diagnosis for" bipolar disorder (applies to other usages of ADO_0000006)
10. Similar issues to #5-6 with CMO:blood measurement.  Not every blood measurement is a clinical marker for bipolar disorder
11. Similar issues to #5-6 with GO:DNA modification. Not every instance of DNA modification is a cellular process for bipolar disorder
12. Similar issues to #5-6 with UBERON:anatomical entity. Not every anatomical entity is an anatomical entity for bipolar disorder.
13. Ontology scope: you have created a number of new disease classes that should go in an OBO disease ontology. This goes against the principle of orthogonality.
14. You did not address the recommendation for fixing Personal Attribute.

BPDO:Author annotation property does not follow naming conventions, does not have proper OBO ID

BPDO:CommentOnDefinition annotation property does not follow naming conventions, does not have proper OBO ID

BPDO:CommentOnHierarchy annotation property does not follow naming conventions, does not have proper OBO ID

BPDO:fromArticle annotation property does not follow naming conventions, does not have proper OBO ID

BPDO:fromBipolarDisorderAnInformationGuide annotation property does not follow naming conventions, does not have proper OBO ID

BPDO:fromDatabase annotation property does not follow naming conventions, does not have proper OBO ID

BPDO:fromDOID annotation property does not follow naming conventions, does not have proper OBO ID

BPDO:fromICD10 annotation property does not follow naming conventions, does not have proper OBO ID

BPDO:fromMFOMD annotation property does not follow naming conventions, does not have proper OBO ID

BPDO:fromNCBIBook annotation property does not follow naming conventions, does not have proper OBO ID

BPDO:fromOMIM annotation property does not follow naming conventions, does not have proper OBO ID

BPDO:fromPubMed annotation property does not follow naming conventions, does not have proper OBO ID

BPDO:fromSNOMEDCT annotation property does not follow naming conventions, does not have proper OBO ID

These APs also have no comments on how they are to be used, rationale is unclear
